README.md

```diff
@@ -38,6 +38,8 @@ and answering your questions. Thanks!
 
 - <first name> <last name> [@<username>](https://github.com/<username>)
```

Comment:
You can remove the line `- <first name> <last name> [@<username>](https://github.com/<username>)` and you should be good. :)

Reply:
It has been removed as requested :)